spinta/formats/rdf/commands.py

```diff
@@ -0,0 +1,388 @@
+    prefix: str = None
+):
+    return Cell(value.isoformat(), prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, Time, datetime.time)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: Time,
+    value: datetime.time,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell(value.isoformat(), prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, DateTime, datetime.datetime)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: DateTime,
+    value: datetime.datetime,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell(value.isoformat(), prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, Number, Decimal)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: Number,
+    value: Decimal,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell(value, prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, Ref, (dict, str, type(None)))
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: File,
+    value: NotAvailable,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    super_ = commands.prepare_dtype_for_response[Context, Format, Ref, dict]
+    data = super_(context, fmt, dtype, value, data=data, action=action, select=select)
+    data['prefix'] = prefix
+    return data
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, File, NotAvailable)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: File,
+    value: NotAvailable,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+):
+    super_ = commands.prepare_dtype_for_response[Context, Format, File, NotAvailable]
+    return super_(context, fmt, dtype, value, data=data, action=action, select=select)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, File, dict)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: File,
+    value: Dict[str, Any],
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    if value['_id'] is None:
+        _id = Cell('', prefix=prefix)
+    else:
+        _id = Cell(value['_id'], about=_get_model_link(
+            dtype.prop.model,
+            pk=data['_id'],
+            prop=dtype.prop.name,
+        ), prefix=prefix)
+    if value['_content_type'] is None:
+        _content_type = Cell('', prefix=prefix)
+    else:
+        _content_type = Cell(value['_content_type'], prefix=prefix)
+    return {
+        '_id': _id,
+        '_content_type': _content_type,
+        'prefix': prefix
+    }
```

Comment:
Files should be included as links `rdf:aboult`, unless `_content` is explicitly requested.